### ISSUE TITLE
Polystrips creating odd polys from existing geometry

### ISSUE THREAD

**Issue by Innuksuk** [CGCookie/retopoflow#223]
I was able to replicate this issue twice. First I extruded the existing target object's outer ring outwards, tweaked them to lay the vertices flat on the source object (extruding in edit mode doesn't snap them to surface), enabled polystrips and tried creating a strip outwards and this is what happened:
![polystrips error](https://cloud.githubusercontent.com/assets/17751811/13688590/e1eec3ca-e6f0-11e5-91f8-423998877e13.png)
![polystrips error 2](https://cloud.githubusercontent.com/assets/17751811/13688616/f230b6bc-e6f0-11e5-8c9f-c7b7f1947455.png)
![polystrips error 3](https://cloud.githubusercontent.com/assets/17751811/13688628/fef93126-e6f0-11e5-997a-cca39ece2732.png)

**Comment by Innuksuk** [comment]
The polystrips is a lot more buggy than the last version. Creating strips from an existing mesh tends to struggle quite a bit. I think it's when you load saved blender files and reassign the source object and the target object that messes it up.
 The polystrips started out fine with the strip working as it should, but when I grabbed the junction point to drag it, an error message popped up and the strip lost the strip count, the junction point froze and only the junction handle was able to move leaving the junction point stuck. Then I selected the strip to increase strip count which I was able to do but error messages popped up each time I did.

![polystrips error](https://cloud.githubusercontent.com/assets/17751811/13725446/0e7eed86-e870-11e5-9c03-9f7a5f6d6fb5.png)

**Comment by patmo141** [comment]
can you upload this .blend file and I will see if I can reproduce here.  Thanks!

**Comment by patmo141** [comment]
I seem to get some weirdness too when extruding from existing.  There appears to be some handles in the wrong spot.  I tried to replicate the steps you described

![image](https://cloud.githubusercontent.com/assets/2917743/13800536/79e9ed4a-eb00-11e5-95d6-9d93bf66aa70.png)

**Comment by carter2422** [comment]
I'll try and replicate these to track down further, thanks for the report @Innuksuk.

The handle being in the wrong place has been logged in #225

**Comment by Innuksuk** [comment]
Sorry I didn't have internet for a couple of days. I think I've figured out the main problem that causes these bugs. When you load saved files with retopoflow, you start off with a single modifier stack (Retopoflow Symmetry), and after reassigning the source object and the target object, it continues on with that same modifier stack. But when you enable X-mirroring (after loading saved files and reassigning the source/target objects) and connect a strip to it's mirror point, a new modifier stack appears (Retopoflow Symmetry.001).

![polystrips symmetry](https://cloud.githubusercontent.com/assets/17751811/13872008/7673566a-ecbd-11e5-9434-c1157be48384.png)

**and the new stack**

![polystrips symmetry 001](https://cloud.githubusercontent.com/assets/17751811/13872023/916a3bfa-ecbd-11e5-94ce-421aab7cd4db.png)

I was able to avoid these issues after deleting the newly created modifier stacks...

P.S. These modifier stacks will keep piling up as you continue reassigning the source object and the target object, enabling x-mirror and connecting them. Hope this leads you to the source of the problem.
